client/src/containers/map/popup/index.tsx

```diff
@@ -16,21 +19,26 @@ const PopupContainer = () => {
       latitude={popup.lngLat.lat}
       longitude={popup.lngLat.lng}
       closeOnClick={false}
-      style={{
-        padding: 0,
-      }}
-      maxWidth="300px"
+      closeButton={false}
+      className="z-50 flex w-[360px] flex-col font-serif"
+      maxWidth="360px"
       onClose={() => setPopup(null)}
     >
-      <div className="pointer-events-none absolute left-0 top-0 h-4 w-full bg-gradient-to-b from-white" />
-      <div className="max-h-[49vh] space-y-2.5 overflow-y-auto overflow-x-hidden pr-8 text-slate-800 shadow-[0_20px_15px_rgba(0,0,0,0.1)]">
-        <div className="divide-y divide-slate-200">
+      <div className="p-6 pr-12">
+        <Button
+          size="icon"
+          onClick={() => setPopup(null)}
+          className="absolute right-0 top-0 flex items-center justify-center bg-slate-700 p-0 text-white transition-colors hover:bg-slate-500 disabled:pointer-events-none disabled:opacity-50 data-[state=open]:bg-accent data-[state=open]:text-muted-foreground"
+        >
+          <X className="h-4 w-4" />
+          <span className="sr-only">Close</span>
+        </Button>
+        <div className="space-y-2 divide-y divide-dashed divide-gray-300">
```

Comment:
The acceptance criteria said:
> Only the value of the top-most layer is displayed (if not visually hidden)

…so we should only display the data of one layer.